llm/ext_server/server.cpp

```diff
@@ -2619,6 +2619,14 @@ static void server_params_parse(int argc, char **argv, server_params &sparams, g
         params.kv_overrides.back().key[0] = 0;
     }
 
+#if defined(GGML_USE_CANN)
```

Comment:
llama.cpp can support multi-gpu now.

Reply:
But llama.cpp in ollama has not been updated to a supported version.